Mathlib/Analysis/NormedSpace/ENormedSpace.lean

```diff
@@ -9,14 +9,14 @@ import Mathlib.LinearAlgebra.Basis.VectorSpace
 /-!
 # Extended norm
 
-In this file we define a structure `ENorm 𝕜 V` representing an extended norm (i.e., a norm that can
```

Comment:
Would you like to split this renaming into its own PR? Should this rename come with deprecations?
(For instance, if the rename lands before the new Lean version, one could add the new name after the new version; providing at least some minimal grace period. We don't have a great way of dealing with "rename X to Y, and introduce X with new meaning" changes, right?)